x-pack/legacy/plugins/apm/public/components/shared/ErrorRateAlertTrigger/index.tsx

```diff
@@ -37,15 +38,19 @@ export function ErrorRateAlertTrigger(props: Props) {
     ...alertParams
   };
 
+  const threshold = isFinite(alertParams.threshold)
```

Comment:
Shouldn't we use `params.threshold ` instead of `alertParams. threshold` ?

Reply:
yes! thanks. I see I mixed it up before as well. Fixed.